src/test/java/it/pagopa/transactions/controllers/TransactionsControllerTest.java

```diff
@@ -378,4 +396,13 @@ void shouldReturnProblemJsonWith400OnBadInput() {
                 .expectBody(ProblemJsonDto.class)
                 .value(p -> assertEquals(400, p.getStatus()));
     }
+
+    @Test
+    void shouldReturnErrorCircuitBreakerOpen(){
+
+        ResponseEntity error = transactionsController.openStateHandler().block();
```

Comment:
More bikeshedding: what about making this method private and mocking a controller method (any would do) with an exception?

Reply:
Yes, I'm planning to refactor all tests related to circuit break logic in dedicated PR

Reply:
> Is the testing proxy committed anywhere?

Nope, it's a node app develop with http-proxy lib